ast/annotations_test.go

```diff
@@ -10,6 +10,80 @@ import (
 	"testing"
 )
 
+func TestEntrypointAnnotationScopeRequirements(t *testing.T) {
+	tests := []struct {
+		note        string
+		module      string
+		expectError bool
```

Comment:
Could we also assert that the scope is set as expected?

Reply:
Sure, I'll amend the tests to do that 👍